test/BootstrapperTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Aerys\Test;
+
+use Aerys\Bootstrapper;
+use Aerys\Console;
+use Aerys\Logger;
+use function Amp\resolve;
+use function Amp\wait;
+
+class BootstrapperTest extends \PHPUnit_Framework_TestCase {
+    /**
+     * @test
+     * @expectedException \DomainException
+     * @expectedExceptionMessage No config file found, specify one via the -c switch on command line
+     */
+    public function throwsWithoutConfig() {
```

Comment:
This test has a problem: it may spuriously fail in case you have a config in the default locations.